.github/workflows/dag-push-production.yaml

```diff
@@ -136,6 +136,7 @@ jobs:
             --bucket=${BUCKET} \
             --src-bucket=${SRC_BUCKET} \
             --sdk-image ghcr.io/wolfi-dev/sdk:latest@sha256:ec70f835c3885d831ae1e5ba08425e0ec08ad2e55ed9b46f5dd5faff459e713c \
+            --gcloud-image cgr.dev/chainguard/google-cloud-sdk:latest \
```

Comment:
Can you pin this and let digestabot float it?  This gives us a rollback story when stuff breaks 🙏